### PR TITLE
chore: ito text position

### LINKED ISSUE #4968: [Bug] Button style issue in ITO card
### How to reproduce?

image

### Mask Version:

- [ ] (Browser) Installed from the extension store (version: )
- [ ] (App) Installed from the app store (version: )
- [ ] master branch
- [x] Build of version/commit hash: /dev

(Please write down build command/flags if there is anything special)

### System

 
- [ ] Windows 10
- [x] Mac OS X Catalina
- [ ] Linux: 

### Platform/Browser

 
- [x] Latest Chrome (stable channel)
- [ ] Latest Firefox (stable channel)
- [ ] Android 11
- [ ] iOS 14

### PR DESCRIPTION
**Pull request by zhouhanseng**
closes #4968 

image

image

### CODE REVIEW

**Comment by zhouhanseng** [comment]
@DimensionDev/qa

**Comment by wenluomask** [comment]
Please change text to "You can only claim after 2021-10-23 15:30 (UTC Time)".

**Comment by wenluomask** [comment]
This text should be only displayed between swap time to unlock time(if it exists).
Screen Shot 2021-12-06 at 4 10 55 PM

**Comment by guanbinrui** [comment]
There is a vast blank area but no place for this simple text. 😓

**Comment by zhouhanseng** [comment]
@wenluomask done

**Comment by wenluomask** [comment]
There are two more issues for ITO:
After tx confirmed, message pops out twice;
Screen Shot 2021-12-13 at 11 40 43 PM

ITO's identity is different from current identity;(As same as Lucky drop)
Screen Shot 2021-12-13 at 11 37 29 PM

**Comment by zhouhanseng** [comment]
> ITO's identity is different from current identity;(As same as Lucky drop)

Forget to rebase develop branch.

File: packages/mask/src/plugins/ITO/SNSAdaptor/ITO.tsx [hunk 1]
```diff
@@ -458,16 +461,15 @@ export function ITO(props: ITO_Props) {
...
-                  })
+        const _text = new BigNumber(availability?.swapped ?? '0').isGreaterThan(0)
+            ? t('plugin_ito_your_swapped_amount', {
+                  amount: formatBalance(availability?.swapped ?? 0, token.decimals),
+                  symbol: token.symbol,
+              })
+            : t('plugin_ito_your_claimed_amount', {
+                  amount: formatBalance(tradeInfo?.buyInfo?.amount_bought ?? 0, token.decimals),
+                  symbol: token.symbol,
+              })
```
Comment (UncleBill) [inline packages/mask/src/plugins/ITO/SNSAdaptor/ITO.tsx:469]:
```suggestion
        const _text = new BigNumber(availability?.swapped || 0).isGreaterThan(0)
            ? t('plugin_ito_your_swapped_amount', {
                  amount: formatBalance(availability?.swapped || 0, token.decimals),
                  symbol: token.symbol,
              })
            : t('plugin_ito_your_claimed_amount', {
                  amount: formatBalance(tradeInfo?.buyInfo?.amount_bought || 0, token.decimals),
                  symbol: token.symbol,
              })
```

`||` is safer than `??` here. `||` can circumvent `availability?.swapped === ''`.
Besides, `??` will be compiled into much more code than `||`.

File: packages/mask/src/plugins/ITO/SNSAdaptor/ITO.tsx [hunk 2]
```diff
@@ -509,6 +511,16 @@ export function ITO(props: ITO_Props) {
             <>
                 <Typography variant="body1">{swapResultText}</Typography>
                 {footerEndTime}
+                {hasLockTime &&
+                !isUnlocked &&
+                unlockTime > Date.now() &&
+                new BigNumber(availability?.swapped ?? '0').isGreaterThan(0) ? (
```
Comment (UncleBill) [inline packages/mask/src/plugins/ITO/SNSAdaptor/ITO.tsx:514]:
```suggestion
                new BigNumber(availability?.swapped || 0).isGreaterThan(0) ? (
```
ditto

**Review by UncleBill — DISMISSED** [review]

**Review by Jack-Works — APPROVED** [review]

**Comment by wenluomask** [comment]
Please check Figma again and change the front size.
Screen Shot 2021-12-15 at 12 57 05 AM

**Comment by wenluomask** [comment]
After withdrawal of ITO, the transaction confirmation kept popping out until I refreshed.
https://user-images.githubusercontent.com/85149605/146045915-afea1647-cb48-4bd7-af20-ab9beccd0bd6.mov

**Comment by zhouhanseng** [comment]
@wenluomask fixed

**Comment by wenluomask** [comment]
Checked.

**Review by yanzhihong23 — APPROVED** [review]

**Review by UncleBill — APPROVED** [review]

**Pull request merged**